api/controller/statushistory/pruner.go

```diff
@@ -8,29 +8,23 @@ import (
 	"time"
 
 	"github.com/juju/juju/api/base"
-	"github.com/juju/juju/api/common"
 	"github.com/juju/juju/rpc/params"
 )
 
 // Option is a function that can be used to configure a Client.
 type Option = base.Option
 
-// WithTracer returns an Option that configures the Client to use the
-// supplied tracer.
-var WithTracer = base.WithTracer
```

Comment:
This should be left for now, this is part of the observability work. This should be redundant, but I want to finally decide if we should keep it or remove it as part of ongoing infrastructure work.

Reply:
it's unused but sure, I can leave it there

Reply:
Thanks, I will decide what to do with this in the next couple of pulses.